code/game/objects/structures/burrows.dm

```diff
@@ -599,3 +634,20 @@ percentage is a value in the range 0..1 that determines what portion of this mob
 	//will be quieter and not travel as far
 	playsound(src, soundtype, maintenance ? volume*0.5 : volume, TRUE,maintenance ? -3 : 0)
 
+/obj/structure/burrow/examine()
+	..()
+	if(isSealed && recieving)
+		to_chat(usr, SPAN_WARNING("You can see movement behind cracks. Maybe they can be welded before whatever behind them break through."))
```

Comment:
Broken English again.
My attempt is "You can see something move behind the cracks. You should weld them shut before it breaks through.". Not perfect, but better already.